include/aikido/planner/dart/ConfigurationToTSR.hpp

```diff
@@ -33,7 +37,9 @@ class ConfigurationToTSR : public Problem
       statespace::dart::ConstMetaSkeletonStateSpacePtr stateSpace,
       ::dart::dynamics::ConstMetaSkeletonPtr metaSkeleton,
       ::dart::dynamics::ConstBodyNodePtr endEffectorBodyNode,
-      std::size_t maxSamples,
+      std::size_t maxSamplingTries,
+      std::size_t batchSize,
+      std::size_t maxBatches,
```

Comment:
In the name of not exploding arguments, thoughts about making this default to 1? Since it effectively just provides a convenient way of avoiding a wrapping for/while loop.

Reply:
Defaulting to 1 in Robot.hpp is fine.